package.json

```diff
@@ -154,18 +154,5 @@
     "hooks": {
       "pre-commit": "lint-staged && npm test"
     }
-  },
-  "lint-staged": {
```

Comment:
linting config moved to `.lintstagedrc.js` so that we can do the .eslint ignore properly